client/app/dbaas/dbaas-metrics/token/edit/metrics-token-edit.html

```diff
@@ -1,4 +1,4 @@
-<form id="MetricsTokenEditForm"
+<form id="MetricsTokenEditForm-id"
```

Comment:
I meant that you could remove the `id` as it wasn't used 😃

Reply:
oh!! okay :) got it